src/call.cpp

```diff
@@ -2821,7 +2821,9 @@ bool call::process_incoming(const char* msg, const struct sockaddr_storage* src)
 #ifdef RTP_STREAM
     /* Check if message has a SDP in it; and extract media information. */
     if (!strcmp(get_header_content(msg, "Content-Type:"), "application/sdp") &&
-            hasMedia == 1 && !curmsg->ignoresdp) {
+        hasMedia == 1 && !curmsg->ignoresdp) {
+      const char *ptr = get_header_content(msg, "content-length:");
```

Comment:
Please use "Content-Length" instead of "content-length", like the other users of get_header_content do.

And use the C++-style pointer placement (`char* ptr` instead of  `char *ptr`).